--- a/dlls/combase/roapi.c
+++ b/dlls/combase/roapi.c
@@ -17,6 +17,7 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 #define COBJMACROS
+#define WIN32_NO_STATUS
 #include "objbase.h"
 #include "initguid.h"
 #include "roapi.h"
@@ -24,6 +25,8 @@
 #include "roerrorapi.h"
 #include "winstring.h"
 #include "errhandlingapi.h"
+#undef WIN32_NO_STATUS
+#include "ntstatus.h"
 
 #include "combase_private.h"
 
@@ -31,6 +34,8 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(combase);
 
+#include "rpcbindingcreate-server-catalog.c"
+
 struct activatable_class_data
 {
     ULONG size;
@@ -474,11 +479,7 @@
  */
 HRESULT WINAPI RoGetServerActivatableClasses(HSTRING name, HSTRING **classes, DWORD *count)
 {
-    FIXME("(%p, %p, %p): stub\n", name, classes, count);
-
-    if (count)
-        *count = 0;
-    return S_OK;
+    return xodus_get_server_classes(name, classes, count);
 }
 
 /***********************************************************************
